Extending fims-web-app to provide a UI for the Group microservice

kengneruphine opened the pull request:
Description
For GSoC, I have extended the fims-web-app to provide a web UI for the Group microservice. I have consumed almost all the endpoints from the microservice and two are left which am working on

 What's included?

- Added Create groupDefinition, view groupDefinition and Edit
- Added manage group, update group
- Implemented group command (Activate, Close and Reopen)
- UI for meetings and sign-off meeting 

Test Steps

- Pull
- npm i
- npm run dev

##### Screenshots or link to CodePen/Plunker/JSfiddle

awasum commented:
@kengneruphine 
This commit changes the base urls of the different services:
https://github.com/apache/fineract-cn-fims-web-app/pull/6/commits/0b00edaa8ce823c51661674793902940caeaaf22

People will not always use your cloud instance so please could you revert the file to point to localhost for the service base Url.

Also check vulnerabilities.

After these, I think we can merge.

kengneruphine commented:
Hello,

I have reverted the base Urls for the different services in the
proxy.conf.json file to localhost.

I have also ran

- npm i
- npm audit fix

to fix any vulnerabilities issues and I have updated my PRs

Regards
Ruphine Kengne

On 2 August 2018 at 14:22, Awasum Yannick <notifications@github.com> wrote:

> @kengneruphine <https://github.com/kengneruphine>
> This commit changes the base urls of the different services:
> 0b00eda
> <https://github.com/apache/fineract-cn-fims-web-app/commit/0b00edaa8ce823c51661674793902940caeaaf22>
>
> People will not always use your cloud instance so please could you revert
> the file to point to localhost for the service base Url.
>
> Also check vulnerabilities.
>
> After these, I think we can merge.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/apache/fineract-cn-fims-web-app/pull/6#issuecomment-409924278>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ASdvfe-QRFVsE1w3lgphBki4hMPuF7ckks5uMv0ygaJpZM4Vq2hV>
> .
>
